src/animals/Crocodile.ts

```diff
@@ -11,45 +11,19 @@ export default class Crocodile extends Reptile {
     }
     this.length = length;
   }
+  
+  
+  giveBirth(): [string, number, boolean,string][] {
```

Comment:
what are we returning here ? where is this data explained ? maybe worth putting it in a object or something

Reply:
What do you mean? Why is it necessary

Reply:
he means that all methods like this one that are public should also have the access modifier, so like this `public giveBirth()`, it adds more clarity to the code

Reply:
so i have to add modifiers to all the methods in the app?

Reply:
maybe only add them to the methods you added in this pr, and in the future we can create a task to fix the others and any other similar issues in the code